test/libraries/SafeToL2Setup.spec.ts

```diff
@@ -0,0 +1,245 @@
+import { expect } from "chai";
+import hre, { deployments, ethers } from "hardhat";
+import { getFactory, getSafeL2SingletonContract, getSafeSingletonContract, getSafeWithOwners } from "../utils/setup";
+import { sameHexString } from "../utils/strings";
+import { executeContractCallWithSigners } from "../../src";
+
+type HardhatTraceLog = {
+    depth: number;
+    gas: number;
+    gasCost: number;
+    op: string;
+    pc: number;
+    stack: string[];
+    storage: { [key: string]: string };
+    memory: string;
+};
+
+type HardhatTrace = {
+    failed: boolean;
+    gas: number;
+    returnValue: string;
+    structLogs: HardhatTraceLog[];
+};
+
+describe("SafeToL2Setup", () => {
+    const setupTests = deployments.createFixture(async ({ deployments }) => {
+        await deployments.fixture();
+        const safeToL2SetupLib = await (await hre.ethers.getContractFactory("SafeToL2Setup")).deploy();
```

Comment:
Should this also be deployed as part of the fixture (making sure that it gets deployed with `hardhat deploy`)?

Reply:
I don't have a strong opinion here, but in the past, I added contracts to the deployment scripts when there was some clarity on their audit status and whether they would be a part of the production deployment. There are other contracts that aren't included

Reply:
OK - lets leave if for a separate PR.

We can add all library contracts that we plan on auditing as part of the "Safe Library COntracts" epic, and add them to the deployment scripts as a separate PR.